DougieDev/refactor-fitlit-B
# Jcs friends card

###### What's this PR do?

- Styling updates for community section and media query
- Refactor and utilize new method getTopPerformer to display top user in catagory
- Fix test files and add tests for multi use function --- still need to implement more sad path testing throughout
- Community cards all displaying 
- Fix issue where today was coming back undefined

###### Where should the reviewer start?

- UserRepo.js --getTopPerformer()
- PageController-- community page display 
   - may still be some room for refactoring 
- style.scss-- media query section
   - I know @GreysonElkins has already been looking through there making adjustments
   - So Thank You!

###### How should this be manually tested?

- npm test-- all tests are passing with the exception to one which is skipped currently
- npm start-- check for bugs and analyze site using devTools

###### Any background context you want to provide?

I think everything we are looking to display is now displaying, just need to adjust some stylings to match other parts of page.

###### What are the relevant tickets?

close #90 
close #91 
close #78 
close #79 
close #80 
close #82 
close #86

###### Screenshots (if appropriate)
###### Questions:

I think that gets me all caught up, so please let me know what I can do to help.
